src/raiden_libs/register_service.py

```diff
@@ -20,6 +20,26 @@
 log = structlog.get_logger(__name__)
 
 
+def checked_transact(
+    web3: Web3, service_address: Address, function_call: ContractFunction, task_name: str
+) -> None:
+    log.info(f"Starting: {task_name}")
+    transaction_hash = function_call.transact({"from": service_address})
+    transaction_receipt = web3.eth.waitForTransactionReceipt(transaction_hash)
+    was_successful = transaction_receipt["status"] == 1
+
+    if not was_successful:
+        log.error(
+            f"Failed: {task_name}\nPlease check that you account is funded.",
```

Comment:
you**r** account is funded

But maybe even -> "Please check that you have enough funds" ?